server/lobbyconnection.py

```diff
@@ -298,64 +298,19 @@ async def command_admin(self, message):
-
-                        self._logger.warning('Administrative action: %s closed client for %s with %s ban (Reason: %s)', self.player, player, duration, reason)
-                        async with self._db.acquire() as conn:
-                            try:
-                                result = await conn.execute(
-                                    select([lobby_ban.c.reason]).where(
-                                        and_(
-                                            lobby_ban.c.idUser == message["user_id"],
-                                            lobby_ban.c.expires_at > func.now()
-                                        )
-                                    )
-                                )
-
-                                row = await result.fetchone()
-                                if row:
-                                    ban_fail = row[0]
-                                else:
-                                    if period not in ["SECOND", "DAY", "WEEK", "MONTH"]:
-                                        self._logger.warning('Tried to ban player with invalid period')
-                                        raise ClientError(f"Period '{period}' is not allowed!")
-
-                                    await conn.execute(
-                                        ban.insert().values(
-                                            player_id=player.id,
-                                            author_id=self.player.id,
-                                            reason=reason,
-                                            expires_at=func.date_add(
-                                                func.now(),
-                                                text(f"interval :duration {period}")
-                                            ),
-                                            level='GLOBAL'
-                                        ),
-                                        duration=duration
-                                    )
-
-                            except pymysql.MySQLError as e:
-                                raise ClientError('Your ban attempt upset the database: {}'.format(e))
-                    else:
-                        self._logger.warning('Administrative action: %s closed client for %s', self.player, player)
-                    if player.lobby_connection is not None:
-                        with contextlib.suppress(DisconnectedError):
-                            await player.lobby_connection.kick()
-                    if ban_fail:
-                        raise ClientError("Kicked the player, but he was already banned!")
-
-            elif action == "broadcast":
-                message_text = message.get('message')
-                if not message_text:
-                    return
+                if player and player.lobby_connection is not None:
+                    self._logger.warning('Administrative action: %s closed client for %s', self.player, player)
```

Comment:
Not sure what your logging policy is but I'd say that's info

Reply:
True